org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/persistence/GitService.xtend

```diff
@@ -62,6 +62,17 @@ class GitService {
 		}
 	}
 
+	def Iterable<String> listAllCommittedFiles() {
```

Comment:
There is also `allFilesAsDiff` / `calculateDiffAgainstEmptyTree` which I think do pretty much the same, except for returning a List<DiffEntry>. So maybe those methods could be reconciled somehow?